packages/runtime/src/controls/rich-text.ts

```diff
@@ -0,0 +1,132 @@
+import { IndexSignatureHack } from '../prop-controllers/descriptors'
+import type * as Slate from 'slate'
+import { BuilderEditMode } from '../utils/constants'
+import { PropController } from '../prop-controllers/instances'
+import { OnChangeParam, Editor } from 'slate-react'
+import { BoxModel } from '../box-model'
+
+export type RichTextControlData = IndexSignatureHack<Slate.ValueJSON>
+
+export const RichTextControlType = 'makeswift::controls::rich-text'
+
+type RichTextControlConfig = {
+  label?: string
```

Comment:
I thought that there were multiple panels for the rich text control. Is this label applied as a prefix to all panel labels?

Reply:
Great catch. I am not doing anything with this value.

Reply:
In that case, we should remove it

Reply:
Also, what are we going to do about labels? It might be confusing when having multiple controls?

Reply:
Agreed. I think it will be confusing, but I think it is ok for now. Users won't be able to find this feature unless they dig for it. We can not release it till sub-selection comes out.